koku/api/report/test/aws/test_views.py

```diff
@@ -600,3 +601,27 @@ def test_invalid_aws_category_key(self):
         url = reverse("reports-aws-costs") + "?group_by[aws_categroy:invalid]=value"
         response = self.client.get(url, **self.headers)
         self.assertEqual(response.status_code, status.HTTP_400_BAD_REQUEST)
+
+    def test_units_consistency(self):
+        """Test if provider map used units match the report."""
+        file = "November-2023-None"
+        file_name = f"{file}.csv"
```

Comment:
Not a blocker, but a separate var here here seems a bit redundant when you could just add the file extension to the file_path:

```
file_path = f"./koku/masu/test/data/aws/{file}.csv"
```